examples/basic/multi_agent_rl_example.py

```diff
@@ -49,6 +50,10 @@ def make_pistonball_env(
 gym.register("Sequoia_PistonBall-v4", entry_point=make_pistonball_env)
 
 
+class MARLSB3Wrapper(SB3VecEnvWrapper):
+    def __init__(self, env):
+        super().__init__(env.env.env.env.env.env.env.venv)
```

Comment:
Ouch! :P
Getting the `venv` attribute doesn't work?
If not, then maybe some kind of while loop, like:
```python
while not isinstance(env, WhateverYouWant) and hasattr(env, "env"):
    env = env.env
```
Let's chat soon, I'd like to better understand what's going on here.